src/main/java/org/myrobotlab/math/MapperLinear.java

```diff
@@ -115,80 +99,29 @@ public void setInverted(boolean invert) {
     inverted = invert;
   }
 
-  /*
-   * (non-Javadoc)
-   * 
-   * @see org.myrobotlab.math.MapperInterface#getMinOutput()
-   */
-  @Override
-  public Double getMinOutput() {
-    return minOutput;
-  }
-
-  /*
-   * (non-Javadoc)
-   * 
-   * @see org.myrobotlab.math.MapperInterface#getMaxOutput()
-   */
-  @Override
-  public Double getMaxOutput() {
-    return maxOutput;
-  }
-
-  /*
-   * (non-Javadoc)
-   * 
-   * @see org.myrobotlab.math.MapperInterface#setMinMaxOutput(double, double)
-   */
-  @Override
-  public void setMinMaxOutput(Double min, Double max) {
-    if (min != null && max != null && max < min) {
-      double t = min;
-      min = max;
-      max = t;
-    }
-    minOutput = min;
-    maxOutput = max;
-  }
-
-  /*
-   * (non-Javadoc)
-   * 
-   * @see org.myrobotlab.math.MapperInterface#getMinInput()
-   */
   @Override
-  public Double getMinInput() {
-    return minInput;
+  public Double getMin() {
+    return this.min;
   }
 
-  /*
-   * (non-Javadoc)
-   * 
-   * @see org.myrobotlab.math.MapperInterface#getMaxInput()
-   */
   @Override
-  public Double getMaxInput() {
-    return maxInput;
+  public Double getMax() {
+    return this.max;
   }
 
-  /*
-   * (non-Javadoc)
-   * 
-   * @see org.myrobotlab.math.MapperInterface#setMinMaxInput(double, double)
-   */
   @Override
-  public void setMinMaxInput(Double min, Double max) {
+  public void setLimits(Double min, Double max) {

```

Comment:
Are we no longer caring out input limits vs output limits?

Reply:
no the world has become much more simple